### ISSUE TITLE
Unable to map numeric shortcut to a another key

### ISSUE THREAD

**Issue by vortexnix** [kdheepak/taskwarrior-tui#527]
**Describe the bug**
In my configuration, shortcut 1 is mapped to a custom script:
`uda.taskwarrior-tui.shortcuts.1=/path/to/custom_script.py`
This works. However, I am unable to remap the shortcut key `1` to `b`or any other key. This does not work:
`uda.taskwarrior-tui.keyconfig.shortcut1=b`

**Environment**
- Operating System: 
```
LSB Version:    core-11.1.0ubuntu2-noarch:printing-11.1.0ubuntu2-noarch:security-11.1.0ubuntu2-noarch
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04
Codename:       focal
```

- Installation: 
Binary from github
`taskwarrior-tui-x86_64-unknown-linux-musl.tar.gz`
`taskwarrior-tui 0.25.4`
`task 2.6.1`

I am sure I am missing something basic here. Kindly help.

**Comment by fullstopslash** [comment]
I've started experiencing this as well.

**Comment by kdheepak** [comment]
This could potentially be related to this issue:

https://github.com/kdheepak/taskwarrior-tui/issues/517

I’ll have to look into it over the weekend. Can you use the previous version until then?

**Comment by vortexnix** [comment]
Sure. No rush. Thanks for looking into it.

**Comment by matt-snider** [comment]
I just ran into this too.

@kdheepak It looks to me like those values simply are [not loaded from the config](https://github.com/kdheepak/taskwarrior-tui/blob/13246297e5b2c549c3a43159f8a3d32140b80a92/src/keyconfig.rs#L127).

i.e. something like

```rust
    let shortcut0 = Self::get_config("uda.taskwarrior-tui.keyconfig.shortcut0", data);
    // ...
    self.shortcut0 = shortcut0.unwrap_or(self.shortcut0);
```

**Comment by kdheepak** [comment]
You are right! Wow, I could have sworn I had implemented that. 

Interested in making a PR? If not, I can fix it the next time I'm at my personal computer.

**Comment by kdheepak** [comment]
I misunderstood the ask here. Sorry for the confusion. This was never implemented to begin with, an artifact of the fact that key configuration was adding after shortcuts were introduced.

Currently it is only possible to have shortcuts bound to 1-9 keys. I'll unable different key shortcuts as part of the refactor/rewrite. Here's the tracking issue: https://github.com/kdheepak/taskwarrior-tui/issues/349

**Comment by matt-snider** [comment]
@kdheepak Ahh ok I see! Thanks for the update.

But wouldn't it be as simple as adding the additional lines for the shortcut 0-9 in the area I linked above? Just wondering because if that is a quick fix that could be made before the refactoring/rewrite, then it would at least get the shortcuts as documented a bit sooner

**Comment by kdheepak** [comment]
I don't think it is just those additional lines, but similar lines will work.

Currently, shortcuts are mapped like this:

```
uda.taskwarrior-tui.shortcuts.1=~/.config/taskwarrior-tui/shortcut-scripts/add-personal-tag.sh
```

As brought up in https://github.com/kdheepak/taskwarrior-tui/issues/520, I think using the `.taskrc` file for this is a little unwieldy and probably better off with a dedicated configuration file.

**Comment by katekorsaro** [comment]
I've just submitted a PR about this
